### PR TITLE
Write Docs on how to acquire API keys

### PR DESCRIPTION
**Pull request by bwillard**

### CODE REVIEW

File: extensions/data-transfer/portability-data-transfer-flickr/Readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Flickr
+This folder contains the extension implementation for the
+[Flickr](https://www.flickr.com) service.
+
+The implementation is based off of [Flickr4Java](https://github.com/boncey/Flickr4Java).
+
+## Data Supported
```
[the remark this answers is not in the document]
Comment (bwillard) [reply extensions/data-transfer/portability-data-transfer-flickr/Readme.md]: Done.

File: Documentation/Developer.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ limitations under the License.
 * It should prompt you to select the new Copyright profile
 * The copyright should be added to all new files from now on (note: it might be collapsed so not immediately obvious)
 
-## Aquire Keys
+## Acquire Keys
 To run an instance of DTP you need to have API keys from the services you wish to interact with.
 Please see the [aquire keys](Keys.md) document for points on how to do this.
```
Comment (rtannenbaum) [inline Documentation/Developer.md]: sp here too in the link

File: Documentation/Keys.md [hunk 3]
```diff
@@ -13,8 +13,13 @@ for each service they want to interact with. This has many reasons and benifits:
...
 But this does mean that to run an instance of DTP you need to do some leg work
-to aquire and mange keys.  We encourage all provideres to keep up to date instructions
-about how aquire keys in each provider's directory.
+to acquire and manage application credentials.  We encourage all providers to
+keep up to date instructions about how acquire application credentials in
+each provider's directory.
+
+API keys usually are just the OAuth App Key and Secret for your app.  But DTP supports
+arbitrary authorization mechanism, and so the exact form the key is dependent on the
+autorizatin technology each providers uses.
```
Comment (rtannenbaum) [inline Documentation/Keys.md]: sp: authorization

File: Documentation/Keys.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+   Terms of Service.  It is important that each Hosting Provider read, understand
+   and agree to any applicable TOS for each service.
+
+ - **Security** Sharing keys is bad for security.
+
+But this does mean that to run an instance of DTP you need to do some leg work
+to aquire and mange keys.  We encourage all provideres to keep up to date instructions
+about how aquire keys in each provider's directory.
+
+ - [Flickr](../extensions/data-transfer/portability-data-transfer-flickr#keys)
```
[the remark this answers is not in the document]
Comment (rtannenbaum) [reply Documentation/Keys.md]: I think the link references needs to be updated from #keys too

File: Documentation/Keys.md (hunk 3 above)
Comment (bwillard) [reply Documentation/Keys.md]: Done

**Pull request merged**